gc/verbose/schema.xsd

```diff
@@ -362,6 +364,9 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 	</complexType>
 
 	<complexType name="gc-start">
+		<sequence maxOccurs="1" minOccurs="0">
+			<element ref="vgc:cpu-util" maxOccurs="1" minOccurs="0" />
+		</sequence>
 		<sequence maxOccurs="1" minOccurs="1">
 			<element ref="vgc:mem-info" maxOccurs="1" minOccurs="0" />
 		</sequence>
```

Comment:
An OpenJ9 internal test (starting with build 75102) doesn't appear to like two `<sequence>` elements here. Perhaps this should be:
```
		<sequence maxOccurs="1" minOccurs="0">
			<element ref="vgc:cpu-util" maxOccurs="1" minOccurs="0" />
			<element ref="vgc:mem-info" maxOccurs="1" minOccurs="0" />
		</sequence>
```